docs/details/commands/pmm-admin.md

```diff
@@ -139,8 +139,8 @@ PMM communicates with the PMM Server via a PMM agent process.
 
     `--metrics-mode=mode`
     : Metrics flow mode for agents node-exporter. Allowed values:
-        - `auto`: chosen by server (default)
-        - `push`: agent will push metrics
+        - `auto`: chosen by server
```

Comment:
`auto` is still default one.

Reply:
on the server side we chose push, but on pmm-admin side `auto` is still the default value.

Reply:
Then nothing changes in pmm-admin man page? (Every is 'auto' default).

Reply:
yes, we can close this PR.